django/contrib/contenttypes/fields.py

```diff
@@ -236,21 +218,29 @@ def __get__(self, instance, cls=None):
         if instance is None:
             return self
 
+        # Make sure to use ContentType.objects.get_for_id() to ensure that
```

Comment:
I wonder if you understand this comment. Is it telling Django's users to prefer get_for_id() over this descriptor method?